modules/swagger-codegen/src/main/resources/qt5cpp/model.mustache

```diff
@@ -38,13 +38,13 @@ public:
 
     void cleanup();
 
-    String asJson ();
+    String asJson () override;
```

Comment:
I dont think this file is actually needed, it doesnt use anything from QT and has a tizen namespace defined. Can you try generating a qt client without this mustache file? (Also remove it from the java file, if necessary)

Reply:
Yes indeed. I just removed it.